pkg/daemon/ceph/client/crush_rule.go

```diff
@@ -37,6 +37,18 @@ rule %s {
         step chooseleaf firstn 2 type %s
         step emit
 }
+`
+	twoStepCRUSHRuleTemplate2 = `
```

Comment:
Instead of duplicating the template, how about modifying one line of the original template like this?
```
step take %s %s
```
Then in the code we have 
```go
var crushRuleInsert string
if pool.DeviceClass != "" {
  crushRuleInsert = fmt.Sprintf("class %s", pool.DeviceClass)
}
```
Then insert that into the other crush rule format.

Reply:
Sure. Will fix.